src/test/java/org/eclipse/basyx/testsuite/regression/submodel/metamodel/map/TestSubmodel.java

```diff
@@ -85,4 +87,27 @@ public void testCreateAsFacadePropertyList() {
 	protected ISubmodel getSubmodel() {
 		return submodel;
 	}
+	
+	@Test(expected = IdShortDuplicationException.class)
+	public void checkForExceptionWithDuplicateIdShortInSubmodel() {
+		Map<String, Object> faultySubmodel = createSubmodelWithDuplicateIdShortProperties();
+
+		Submodel.createAsFacade(faultySubmodel);
+	}
+
+	private Map<String, Object> createSubmodelWithDuplicateIdShortProperties() {
+		Property property1 = new Property("testProp", 5);
+		Property property2 = new Property("testProp", 7);
+		
+		Collection<Map<String, Object>> collection = new ArrayList<>();
+		
+		collection.add(property1);
+		collection.add(property2);
+		
+		Submodel submodel = new Submodel();
```

Comment:
Please add an arbitrary idShort & identifier to ensure constructing a valid submodel

Reply:
Added an idShort & identifier.
Updated the code.